### PR TITLE
README update for DELG training

### PR DESCRIPTION
**Pull request by dan-anghel**
# Description

README update for training DELG global feature.

## Type of change

- [x] Documentation update

## Tests

- N/A

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: research/delf/delf/python/training/README.md [hunk 1]
```diff
@@ -164,6 +164,22 @@ the batch size to `256`:
...
+python3 train.py \
+  --train_file_pattern=gldv2_dataset/tfrecord/train* \
+  --validation_file_pattern=gldv2_dataset/tfrecord/validation* \
+  --imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 \
+  --dataset_version=gld_v2_clean \
+  --logdir=gldv2_training/ \
+  --delg_global_features
+```
+
+
```
Comment (andrefaraujo) [inline research/delf/delf/python/training/README.md:182]: quick sanity check, will the model trained in this manner work with the export commands below? Maybe there needs to be some edit? (eg, I think the global exporting script would need some arguments to denote that new pooling/embedding layers would be used.)

Comment (dan-anghel) [reply research/delf/delf/python/training/README.md:182]: Thanks for the feedback André! I will check for backwards compatibility and update the PR accordingly.

Comment (dan-anghel) [reply research/delf/delf/python/training/README.md:182]: I have successfully tested 1) exporting the DELF local feature model with the newly trained model, 2) extracting the DELF features for 2 test images and 3) matching based on the extracted features and it works as before. Also extracting the Kaggle-compatible global feature model works with the newly trained model, however the script instantiates a Delf() model not a Delg() model. There are indeed changes to be made to the global feature model script (examples/export_global_model.py) in order to allow for exporting a Delg() model. These changes being Python code I would need to make a new commit. Please let me know if you want me to mix the Python changes with this README update in the same PR or if I should make a separate PR with the Python code only (and potentially some additional README update). Thank you!

Comment (andrefaraujo) [reply research/delf/delf/python/training/README.md:182]: Thanks Dan! Yes, I agree we can do those changes in a separate commit. Let me merge this PR now.

**Pull request merged**